fluentd/http/values.yaml

```diff
@@ -174,7 +174,6 @@ fluentd:
           applicationName ${record.dig("kubernetes", "#{ENV['APP_NAME']}")}
           subsystemName ${record.dig("kubernetes", "#{ENV['SUB_SYSTEM']}")}
           computerName ${record.dig("kubernetes", "host")}
-          timestamp ${time.strftime('%s%L')}
```

Comment:
why do we need to delete it ?

Reply:
we were always using the timestamp field , its was always a part of the records being sent no ? Im not sure I understand